### ISSUE TITLE
show last content when opening app when no new data can be fetched

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#182]
The app opens sometimes in such a state (without the "no data connection" pop-up appearing):
![Screenshot_20191003-144317](https://user-images.githubusercontent.com/323719/66127866-f8c1be00-e5ec-11e9-9ae2-5b9a1d72b944.png)

This happens, e.g. on the move in a train/metro with network coming and going or when a device is connected to a router, but the router is not connected to the internet.

In such a case, let's always display the most recent data from cache with indeed the corresponding last update timestamp, but also a little spinning wheel (e.g. left of the location title) indicating that the app is still trying to fetch fresh data.
Bottom line, always display some data..

**Comment by SpeckiJ** [comment]
The behaviour of the cache has changed with commit https://github.com/irceline/aq-mobile-be/commit/50ddd2e6b5a84b2d66e771ea680b3ad000741d5e relating to https://github.com/irceline/aq-mobile-be/issues/109

The local cache is now only cleared if there is an active network connection. As long as there is valid cached data (tiles are cached 24h, data is cached ??hours) this data is now displayed.

> but also a little spinning wheel (e.g. left of the location title) indicating that the app is still trying to fetch fresh data.

As noted in https://github.com/irceline/aq-mobile-be/issues/109 this is really necessary. I would even propose a big red notification icon (e.g. [something like this](https://www.google.com/search?q=red+alert+icon&client=firefox-b-d&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjqh9ix97TlAhXBDmMBHbaoCIoQ_AUIESgB&biw=1888&bih=947) that indicates that the app is currently offline.

**Comment by opeeters** [comment]
Agreed with proposal to display red alert icon. Could also be (dark) yellow, but maybe add some animation to it as well.

**Comment by SpeckiJ** [comment]
I have added the warning in https://github.com/irceline/aq-mobile-be/commit/0ee1d57d363e2c7373056eb96b843e6c35f34b19

This is still missing a translations: 
https://github.com/irceline/aq-mobile-be/blob/0ee1d57d363e2c7373056eb96b843e6c35f34b19/src/assets/i18n/fr.json#L329

https://github.com/irceline/aq-mobile-be/blob/0ee1d57d363e2c7373056eb96b843e6c35f34b19/src/assets/i18n/nl.json#L329

@opeeters Can you add the missing translations?

**Comment by opeeters** [comment]
translations were added with 2c41bc8247cc903ec1623c7d8da0779ca94ce9b0
